related_topics/alcohol_consumption/alcohol_dataset.py

```diff
@@ -0,0 +1,73 @@
+import kagglehub
+import pandas as pd
+from typing import Dict, Type, List
+import re
+import os
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+
+class AlcoholDataset:
+    _KAGGLE_DATASET_PATH: str = (
+        "thedevastator/oecd-alcohol-consumption-per-capita/versions/2"
+    )
+    _FILE_YEAR_PATTERN = re.compile(r"WHR_(\d{4}).csv")
+    _YEARS = ['2015', '2016', '2017', '2018', '2019', '2020', '2021', '2022']
+
+    def __init__(
+        self, year_to_alchohol_time_series: Dict[str, pd.DataFrame]
+    ) -> None:
+        self._alchohol_time_series = year_to_alchohol_time_series
+        self._years = sorted(
+            list(year_to_alchohol_time_series.keys())
+        )
+
+    def get_years(self) -> List[str]:
+        return self._years
+
+    def __getitem__(self, year: int) -> pd.DataFrame:
+        year_str = str(year)
+        if re.fullmatch(r"\d{2}", year_str) is not None:
+            year_str = f"20{year_str}"
+
+        assert re.fullmatch(r"\d{4}", year_str) is not None, (
+            "Year must be a 4-digit number or a 2-digit number that "
+            "can be converted to a 4-digit number by prepending '20'."
+            f" Got {year_str}."
+        )
+        return self._alchohol_time_series[year_str]
```

Comment:
I didn't know we intended to download all data.
On telegram, I literally just meant adding the links to README, so we can keep track of the potential datasets.

Reply:
I think this will make it easier to explore datasets for everyone + having the same style of working with data  (if lets say some people work on different datasets)